sdcm/utils/features.py

```diff
@@ -75,8 +76,9 @@ def is_consistent_topology_changes_feature_enabled(session: Session) -> bool:
     return CONSISTENT_TOPOLOGY_CHANGES_FEATURE in get_enabled_features(session)
 
 
-def is_tablets_feature_enabled(session: Session) -> bool:
+def is_tablets_feature_enabled(node: BaseNode) -> bool:
```

Comment:
Would it be possible to get any node from session instead?

Reply:
In current use cases, it is easy to get a node. Using session here would mean that I need to create a session first, then return node (which I already have).